base_layer/wallet_ffi/src/lib.rs

```diff
@@ -1874,22 +1944,22 @@ pub unsafe extern "C" fn wallet_test_mined(
     }
 }
 
-/// This function simulates the detection of a `TariPendingInboundTransaction` as being broadcast
-/// to base layer which means the Pending transaction must become a `TariCompletedTransaction` with
-/// the `Broadcast` status.
+/// This function will simulate the process when a completed transaction is detected as mined on
+/// the base layer. The function will update the status of the completed transaction AND complete
+/// the transaction on the Output Manager Service which will update the status of the outputs
 ///
 /// ## Arguments
 /// `wallet` - The TariWallet pointer
-/// `tx` - The TariPendingInboundTransaction pointer
+/// `tx_id` - The TxId of the transaction to operate on
 /// `error_out` - Pointer to an int which will be modified to an error code should one occur, may not be null. Functions
 /// as an out parameter.
 ///
 /// ## Returns
 /// `bool` - Returns if successful or not
 #[no_mangle]
-pub unsafe extern "C" fn wallet_test_transaction_broadcast(
+pub unsafe extern "C" fn wallet_test_mine_transaction(
     wallet: *mut TariWallet,
-    tx: *mut TariPendingInboundTransaction,
+    tx_id: c_ulonglong,
```

Comment:
Consider a pointer a transaction struct instead of c_ulonglong for the same reasons as above

Reply:
done